package.json

```diff
@@ -15,12 +15,18 @@
     "test": "npm run test:unit && npm run test:e2e"
   },
   "dependencies": {
+    "@auth0/auth0-spa-js": "^1.14.0",
     "@vue/babel-preset-app": "^4.5.10",
+    "axios-request-handler": "^1.0.4",
+    "codecov": "^3.7.0",
     "compression-webpack-plugin": "^6.0.4",
+    "flush-promises": "^1.0.2",
     "idb": "^6.0.0",
     "konva": "^7.1.4",
     "moment": "^2.26.0",
+    "moment-range": "^4.0.2",
```

Comment:
Where is `moment-range` used in code?

Reply:
It was been used previously to determine if a subscription was active using the `duration` object. This has been changed. I have uninstalled the library.